src/relay/op/op_common.h

```diff
@@ -60,6 +60,7 @@ namespace relay {
       .add_type_rel("Identity", IdentityRel)                                   \
       .set_attr<TOpPattern>("TOpPattern", kElemWise)                           \
       .set_attr<TOpIsStateful>("TOpIsStateful", false)                         \
+      .set_attr<CommutativeOp>("CommutativeOp", false)                         \
```

Comment:
well, I guess so...

Reply:
Do we need to put the commutativity property on unary and ops with 3+ arity?
I guess the downside is that then you'd need to check that arity == 2 before looking at the CommutativeOp attribute, or that the CommutativeOp attr is actually on the op.